pkgs/top-level/all-packages.nix

```diff
@@ -27525,9 +27525,8 @@ with pkgs;
 
   merkaartor = libsForQt5.callPackage ../applications/misc/merkaartor { };
 
-  mepo = callPackage ../applications/misc/mepo {
-    zig = zig_0_8_1;
-  };
+  mepo = callPackage ../applications/misc/mepo { };
+  mepo-wayland = callPackage ../applications/misc/mepo { withWayland = true; };
```

Comment:
... to avoid this and people can easily use wayland.

Reply:
This is needed for either Wayland or X11 but I think it's OK to default to Wayland.